crates/revive/src/evm.rs

```diff
@@ -0,0 +1,285 @@
+// This file is part of Substrate.
```

Comment:
This code is copied from `pallet_revive`, so I’ve kept their copyright notice. Not sure if there’s a better practice here.